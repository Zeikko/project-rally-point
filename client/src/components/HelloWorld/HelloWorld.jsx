import React from 'react'
import styled from 'styled-components'

const Heading = styled.h1`
  font-size: 64px;
  text-align: center;
`

function HelloWorld() {
  return (
    <Heading>Hello World!</Heading>
  )
}

export default HelloWorld
