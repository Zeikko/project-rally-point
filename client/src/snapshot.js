import renderer from 'react-test-renderer'
import 'jest-styled-components'

export function matchComponentToSnapshot(component) {
  const json = renderer.create(component).toJSON()
  expect(json).toMatchSnapshot()
}