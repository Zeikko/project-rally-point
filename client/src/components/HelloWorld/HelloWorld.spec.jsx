import React from 'react'
import HelloWorld from './HelloWorld'
import { matchComponentToSnapshot } from '../../snapshot'

describe('HelloWorld', () => {
  it('renders correctly', () => {
    matchComponentToSnapshot(<HelloWorld />)
  })
})