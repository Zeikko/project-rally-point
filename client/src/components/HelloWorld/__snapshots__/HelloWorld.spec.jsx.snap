// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld renders correctly 1`] = `
.c0 {
  font-size: 64px;
  text-align: center;
}

<h1
  className="c0"
>
  Hello World!
</h1>
`;
